### PR TITLE
PXB-2787 Add info about Amazon KMS to docs

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: source/advanced/encrypted_innodb_tablespace_backups.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 1]
```diff
@@ -13,14 +13,18 @@ master encryption key is stored in a keyring. |xtrabackup| supports two keyring
 plugins: ``keyring_file``, and ``keyring_vault``. These plugins are installed
 into the ``plugin`` directory.
 
-Implemented in |xtrabackup| version 8.0.25, adds support for the ``keyring_file`` component, which is part of the component-based infrastructure MySQL includes to extend the server capabilities. The component is stored in the ``plugin`` directory. 
+Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is an encryption and key management service used in the cloud. The keys and functionality can be used for other AWS services or your applications that use AWS. 
 
-See a `comparison of keyring components and keyring plugins <https://dev.mysql.com/doc/refman/8.0/en/keyring-component-plugin-comparison.html>`__ for more information.
+No configuration is required to back up a server with AWS KMS-enabled encryption.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Add a note about:

This feature is *tech preview* quality.

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 2]
```diff
@@ -13,14 +13,18 @@ master encryption key is stored in a keyring. |xtrabackup| supports two keyring
...
 
 Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server.
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Add a note about:

No configuration is required to back up a server with AWS KMS-enabled encryption.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: updated

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: updated

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst [hunk 3]
```diff
@@ -21,6 +21,12 @@ Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperabilit
 
 This feature is *tech preview* quality.
 
+Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is an encryption and key management service used in the cloud. The keys and functionality can be used for other AWS services or your applications that use AWS.
```
Comment (patrickbirch) [inline storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Can we fix all three of the references?
Percona XtraBackup 8.0.25-17  adds support for the keyring_file component, which is part of the component-based infrastructure MySQL which extends the server capabilities. The component is stored in the plugin directory.
Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server. This feature is tech preview quality.
Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is a cloud-based encryption and key management service. The keys and functionality can be used for other AWS services or your applications that use AWS. This feature is tech preview quality. (edited)

**Review by altmannmarcelo — APPROVED** [review]
after @patrickbirch 's suggestions - LGTM

File: storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst (hunk 3 above)
Comment (alina-derkach-oaza) [reply storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst]: Updated

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**